xs/Buffer.xs

```diff
@@ -0,0 +1,53 @@
+MODULE = FFI::Platypus PACKAGE = FFI::Platypus::Buffer
+
+# Stolen from Data::Peek::DGrow
+void
+grow (sv, size, ... )
+    SV     *sv
+    IV      size
+ 
+  PROTOTYPE: $$;$
+  PREINIT:
+    int clear = 1;  
+  PPCODE:
+    if ( items > 2 )
+        clear = SvIV(ST(2));
+
+    if (SvROK (sv))
+        sv = SvRV (sv);
+    if (!SvPOK (sv))
+        sv_setpvn (sv, "", 0);
```

Comment:
if `!SvPOK(sv)` and `PERL_API_VERSION < 26` then we `sv_setpvn (sv, "", 0);` twice.

If fact I think that is you `sv_setpvn(sv, "", 0);` you don't need either version of clear?

Reply:
I cargo cult copied some of that; I'll have to regain some of my XS-fu to see why it's done in Data::Peek that way.  In any case, we should use the modern version of clear when possible as its more efficient.

Reply:
see https://github.com/Perl5-FFI/FFI-Platypus/pull/225#pullrequestreview-358291734